helpers/resourceHints.js

```diff
@@ -10,27 +16,38 @@ const fontResources = {
     ],
 };
 
+function format(host) {
+    return `<link rel="dns-prefetch preconnect" href="${host}" crossorigin>`;
+}
+
 const factory = globals => {
-    return function() {
-        function format(host) {
-            return `<link rel="dns-prefetch preconnect" href="${host}" crossorigin>`;
-        }
+    return function () {
 
-        var hosts = [];
+        let hosts = [];
```

Comment:
it's `const`

Reply:
maybe it is not because we are reassigning it at line [39](https://github.com/bigcommerce/paper-handlebars/pull/188/files#diff-b2e5ac345eb2c02e97a25341cda5e936b4952130cfa11ba6e10e44d9479ca03dR39)